src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReportTest.java

```diff
@@ -196,14 +196,14 @@ public void testConsiderCaseIfValidHearingStatusReportSummary() {
 
         List<AdhocReportTypeItem> adhocReportTypeItemList = reportListingData.getLocalReportsSummary();
         AdhocReportType adhocReportType = adhocReportTypeItemList.get(0).getValue();
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getTotal())?0:Integer.parseInt(adhocReportType.getTotal()));
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getHearing())?0:Integer.parseInt(adhocReportType.getHearing()));
-        assertEquals(0, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getHearingCM())?0:Integer.parseInt(adhocReportType.getHearingCM()));
-        assertEquals(0, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getCosts())?0:Integer.parseInt(adhocReportType.getCosts()));
-        assertEquals(0, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getHearingPrelim())?0:Integer.parseInt(adhocReportType.getHearingPrelim()));
-        assertEquals(0, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getReconsider())?0:Integer.parseInt(adhocReportType.getReconsider()));
-        assertEquals(0, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getRemedy())?0:Integer.parseInt(adhocReportType.getRemedy()));
-        assertEquals("1970-06-01T00:00:00.000", adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getDate())?0:adhocReportType.getDate());
+        assertEquals(1, Strings.isNullOrEmpty(adhocReportType.getTotal())?0:Integer.parseInt(adhocReportType.getTotal()));
+        assertEquals(1, Strings.isNullOrEmpty(adhocReportType.getHearing())?0:Integer.parseInt(adhocReportType.getHearing()));
+        assertEquals(0, Strings.isNullOrEmpty(adhocReportType.getHearingCM())?0:Integer.parseInt(adhocReportType.getHearingCM()));
+        assertEquals(0, Strings.isNullOrEmpty(adhocReportType.getCosts())?0:Integer.parseInt(adhocReportType.getCosts()));
+        assertEquals(0, Strings.isNullOrEmpty(adhocReportType.getHearingPrelim())?0:Integer.parseInt(adhocReportType.getHearingPrelim()));
+        assertEquals(0, Strings.isNullOrEmpty(adhocReportType.getReconsider())?0:Integer.parseInt(adhocReportType.getReconsider()));
+        assertEquals(0, Strings.isNullOrEmpty(adhocReportType.getRemedy())?0:Integer.parseInt(adhocReportType.getRemedy()));
+        assertEquals("1970-06-01T00:00:00.000", Strings.isNullOrEmpty(adhocReportType.getDate())?0:adhocReportType.getDate());
```

Comment:
why is the null check necessary? if it's null and the test is expecting a value it'll fail the assertion?